bin/runner.js

```diff
@@ -19,7 +19,9 @@ try {
 
         throw err;
       })
-      .then(() => runner.runTests())
+      .then(async () => {
+        argv.sequential ? await runner.runTests(): runner.runTests();
```

Comment:
can you explain the logic here? keep in mind that a .then() will always return a promise.

Reply:
Actually I'd want to make a chain of promises on `runner.runTests()` to run each environment sequentially but the issue is everything (also environments) is already setup by `setupAsync` function. So for that reason I created a promises chain of `callback({...argv, env: env, e: env})`  and started waiting for `runner.runTests()` to complete. Once `runner.runTests` completes its job then only I'll pick the next `callback({...argv, env: env, e: env})`. 

Because I was only waiting in sequential case this is why I have added this condition.

Reply:
I think this change is problematic. Can you try reverting it?